src/lib/unifyPageContent.tsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import { renderToString } from 'react-dom/server';
+
+export const unifyPageContent = ({
+    css,
+    js,
+    html,
+}: {
+    css?: string;
+    js: string;
+    html?: string;
+}) =>
+    renderToString(
+        <>
+            {`<!DOCTYPE html>`}
+            <html>
+                <head>
+                    <meta charSet="utf-8" />
+                    <meta
+                        name="viewport"
+                        content="width=device-width,minimum-scale=1,initial-scale=1"
+                    />
```

Comment:
Adding fonts from parent frame

```suggestion
                    />
                    <script>
                    var fonts = [].slice.apply(window.parent.document.styleSheets)
					    .filter(function (sheet) { return sheet.ownerNode.className.indexOf("webfont") > - 1; })
					    .map(function (sheet) { return sheet.ownerNode.textContent; })
					    .join(' ');
					var css = document.createElement('style');
					css.textContent = fonts;
					document.head.appendChild(css);
                    </script>
```